config/manager/manager.yaml

```diff
@@ -102,3 +102,26 @@ spec:
       - name: webhook-tls
         secret:
           secretName: webhook-server-tls
+---
+kind: PodDisruptionBudget
+metadata:
+  name: controller-manager-pdb
+  namespace: default
+spec:
+  minAvailable: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+      app.kubernetes.io/name: k8s-pdb-autoscaler
+      app.kubernetes.io/managed-by: kustomize
+---
+apiVersion: apps.mydomain.com/v1
+kind: PDBWatcher
```

Comment:
So a way to test is to use install.sh to toss this on a cluster then use go run ./cmd/evict --ns default --pod <your controllers name> 
First one should fail but you should get two and then if you run it again it should succeeed and scale you back to 1.

Reply:
it worked 


default             controller-manager-6d89fcb548-b8wqv      0/1     Terminating         0                2m9s    10.244.1.105   aks-nodepool1-12289360-vmss000000   <none>           <none>
default             controller-manager-6d89fcb548-b8wqv      0/1     Terminating         0                2m9s    10.244.1.105   aks-nodepool1-12289360-vmss000000   <none>           <none>
default             controller-manager-6d89fcb548-jxhsh      0/1     Running             0                1s      10.244.0.217   aks-nodepool1-12289360-vmss000001   <none>           <none>